File: 03_MOBILE/ios/TerminalAttach/TerminalEventEmitter.swift
import Foundation

public final class TerminalEventEmitter {
  public private(set) var lastName: String?
  public private(set) var lastBody: [String: Any]?

  public init() {}

  public func send(name: String, body: [String: Any]) {
    lastName = name
    lastBody = body
  }
}

File: 03_MOBILE/android/app/src/main/java/com/stratoc/mobile/terminal/TerminalEventEmitter.kt
package com.stratoc.mobile.terminal

class TerminalEventEmitter {
    var lastName: String? = null
        private set
    var lastBody: Map<String, Any?>? = null
        private set

    fun sendEvent(name: String, body: Map<String, Any?>) {
        lastName = name
        lastBody = body
    }
}
